### DISCUSSION TITLE
Deployments failing with pnpm

### DISCUSSION THREAD

**Discussion by Mitsunee** [vercel/community#2646] [Help] [answered]
### Summary

pnpm just updated and my lockfile is now using lockfile version 6.1. This has caused my deployments to fail with lockfile errors claiming a dependency I didn't even update hasn't been found. What can I do to ensure vercel uses an up-to-date version of pnpm?

### Steps to Reproduce

- upgrade to latest version of pnpm locally `npm i -g pnpm@latest`
- run `pnpm i` to update the lockfile
- push lockfile to github

**Comment by joulev** [comment, marked as answer]
Yeah it appears Vercel haven't upgrade their pnpm version yet. In the meantime you can go to #2641 for a workaround.

**Comment by joulev** [comment]
Update: it has been fixed.
